Tests/SwiftStdlibTests/KeyedDecodingContainerTests.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  KeyedDecodingContainerTests.swift
+//  SwifterSwift
+//
+//  Created by Francesco Deliro on 23/10/2019.
+//  Copyright © 2019 SwifterSwift
+//
+
+import XCTest
+
+@testable import SwifterSwift
+
+private struct Video: Decodable {
+    var isPlaying: Bool
+    var isFullScreen: Bool?
+
+    enum CodingKeys: String, CodingKey {
+        case isPlaying
+        case isFullScreen
+    }
+
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        isPlaying = try container.decodeBoolAsIntOrString(forKey: .isPlaying)
+        isFullScreen = try container.decodeBoolAsIntOrStringIfPresent(forKey: .isFullScreen)
+    }
+}
+
+final class KeyedDecodingContainerTests: XCTestCase {
+
+    private let isPlayingAndIsFullScreenAsInt = #"{"isPlaying": 1, "isFullScreen": 0}"#
+    private let isPlayingAndIsFullScreenAsString = #"{"isPlaying": "true", "isFullScreen": "false"}"#
+    private let isPlayingAndIsFullScreenAsBool = #"{"isPlaying": true, "isFullScreen": false}"#
+    private let isPlayingAsIntAndIsFullScreenNotPresent = #"{"isPlaying": 0}"#
+    private let isPlayingAndIsFullScreenTypeMismatch = #"{"isPlaying": [1], "isFullScreen": ["true"]}"#
+
+    func testDecodeBoolAsIntOrStringDataAsIntSuccessful() {
+        let data = mockJsonData(from: isPlayingAndIsFullScreenAsInt)
+        let result = try? JSONDecoder().decode(Video.self, from: data)
```

Comment:
```suggestion
        let result = try! JSONDecoder().decode(Video.self, from: data) // swiftlint:disable:this force_try
```
Not sure if the `swiftlint` call is necessary (check without), but you know that this is going to work, so save yourself the extra code for all these functions.